Events/BodyScannedEvent.cs

```diff
@@ -47,13 +47,15 @@ static BodyScannedEvent()
 
         public string name { get; private set; }
 
+        public string planettype { get { return bodyclass; } } // This is the object property reported from the BodyDetails() function
```

Comment:
Can be more concisely written as `public string planettype => bodyclass; // ...`

VS can do this for you: hover in the left margin to the right of the line number until the lightbulb appears, then click, `Alt`+`Enter`, or `Ctrl`+`.`.